packages/bundler-plugin-core/src/index.ts

```diff
@@ -317,6 +323,22 @@ export function sentryUnpluginFactory({
       );
     }
 
+    if (!options.componentNameAnnotate) {
+      logger.warn(
+        "No options provided for the component name annotate plugin. Please set `componentNameAnnotatePlugin.enabled` to `true` if you would like to have your frontend components automatically annotated at build-time."
+      );
+    } else if (!options.componentNameAnnotate.enabled) {
```

Comment:
We should not print this log message generically. It wouldn't make any sense in non-react applications.

Reply:
True, I will remove it in that case then